src/screens/FeedScreen/Content.js

```diff
@@ -339,14 +340,14 @@ export const styles = StyleSheet.create({
   feedDate: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.black,
+    color: colors.black,
     lineHeight: 18
   },
   point: {
     width: 4,
     height: 4,
     borderRadius: 4,
-    backgroundColor: COLORS.gray8,
+    backgroundColor: colors.gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated correctly from `COLORS.*` to `colors.*`. However, the background color value in line 350 has changed from `COLORS.gray8` to `colors.gray`. Ensure that this change is intentional and that the new color value (`colors.gray`) exists in the `colors` module.
```diff
-    backgroundColor: COLORS.gray8,
+    backgroundColor: colors.gray,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->